### ISSUE TITLE
Not necessary tabs

### ISSUE THREAD

**Issue by iJahangard** [aistra0528/Hail#111]
As long as hail is a simple lightweight app with amazing feature of freezing apps and manage them
Its UI should psychologically represent a simplicity

We have four tabs at the bottom which two of them are barley being used by user, settings and about
Settings should be an icon somewhere and about can be a menu in settings

We have apps tab which it also can being shown as an icon anywhere (like as a sub icon upper than freeze button)
Also Tags will be more accessible if they are arranged in button of the page i guess

**Comment by tuyafeng** [comment]
I totally agree with your point of view. I once submitted a PR to remove the navigation bar, which was rejected by the author, but I still hope that the author will seriously consider whether the bottom navigation bar is unnecessary.

**Comment by iJahangard** [comment]
Adding a searchbar instead of those tabs might be a better visualization, search bar at bottom of the page and tags are at top of search bar
And for settings and apps sections it can be two buttons on a row at top of list
For pinned apps it can be an accent color around the app icon which I think it's really handy
If i had time i would design what i mean in figma and share it here
